as/cNFT/assembly/royalty_payout.ts

```diff
@@ -0,0 +1,45 @@
+import { u128 } from 'near-sdk-as'
+import { royalty_to_payout } from '../../utils'
```

Comment:
Here you have imported `royalty_to_payout` method from the `as/utils.js` file in root folder. But you have already added a `royalty_to_payout` method  in `as/cNFT/assembly/utils/royalties.ts` in too. So I think you should import it from that one.

Reply:
Correct!